lucille-admin/src/main/java/com/kmwllc/lucille/objects/RunStatus.java

```diff
@@ -0,0 +1,43 @@
+package com.kmwllc.lucille.objects;
+
+public class RunStatus {
+
+  private final String runId;
+
+  private boolean isRunning;
+
+  public RunStatus(String runId, boolean isRunning) {
+    this.runId = runId;
+    this.isRunning = isRunning;
+  }
+
+  public RunStatus(boolean isRunning) {
+    this.runId = "";
+    this.isRunning = isRunning;
+  }
+
+  public String getRunId() {
+    return runId;
+  }
+
+  public boolean isRunning() {
+    return isRunning;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (!(o instanceof RunStatus)) {
+      return false;
+    }
+
+    RunStatus status = (RunStatus) o;
+
+    return this.isRunning == status.isRunning && this.runId.equals(status.runId);
```

Comment:
if runId can ever be null (i.e. via new RunStatus(null, false)) this would give a nullpointerexception at this.runId.equals()